### DISCUSSION TITLE
Keep getting the error: "The function 'XYZ' doesn't exist, is disabled, or is not an orchestrator function."

### DISCUSSION THREAD

**Discussion by austin-altaml** [Azure/azure-functions-durable-extension#2353] [Q&A]
I'm trying to run Azure Orchestrations and I sunddenly I started getting the error from the statusQueryGetUri:

```Orchestrator function 'UploadDocumentOrchestrator' failed: The function 'UploadDocumentOrchestrator' doesn't exist, is disabled, or is not an orchestrator function.```

I've run this functions dozens of times today without any issue. VSCode doesn't flag any issues. I've tried restarting and that hasn't helped either.

This has happened to meonce before. I got the error, troubleshot for like 3 hours, got frustrated, left, and came back to my computer an hour later and the problem was resolved without changing anything. I suspect the issue is related to caching somehow, but I'm not sure where to fix this.

I'm on Mac. There is a similar issue noted at: https://github.com/Azure/azure-functions-durable-extension/issues/577

After I run func start, my "missing" orchestration appears in the launch screen indicating that it is valid as shown below.

I start the UploadDocumentOrchestrator with the HttpTrigger StartUploadDocuments. I see the trigger complete its execution UploadDocumentOrchestrator never starts in the logs.
```
...

    UploadDocumentOrchestrator: orchestrationTrigger

For detailed output, run func with --verbose flag.
[2023-01-11T19:58:10.490Z] Executing 'StartUploadDocuments' (Reason='This function was programmatically called via the host APIs.', Id=f158ac3e-fa39-403c-9671-307ea54d5948)
[2023-01-11T19:58:10.599Z] Started orchestration with ID = '29b9ab7fcffe4f59bd47032ef21c19e8'.
[2023-01-11T19:58:10.620Z] Executed 'StartUploadDocuments' (Succeeded, Id=f158ac3e-fa39-403c-9671-307ea54d5948, Duration=154ms)
[2023-01-11T19:58:13.952Z] Host lock lease acquired by instance ID '0000000000000000000000004906C298'.
```

**Comment by nytian** [comment]
Hi, thanks for the discussion. I would like to ask some questions to help us investigate. 

First, I assume that you ran the app locally, could you please confirm that for me? And could you tell us how often the problem happens? Besides, I wonder if you could provide a minimal project which had this problem? That would be helpful for us to debug it and find the reason. Thanks.
